
# This is a patch for Task-Catalyst-4.00.orig to update it to Task-Catalyst-4.00
# 
# To apply this patch:
# STEP 1: Chdir to the source directory.
# STEP 2: Run the 'applypatch' program with this patch file as input.
#
# If you do not have 'applypatch', it is part of the 'makepatch' package
# that you can fetch from the Comprehensive Perl Archive Network:
# http://www.perl.com/CPAN/authors/Johan_Vromans/makepatch-x.y.tar.gz
# In the above URL, 'x' should be 2 or higher.
#
# To apply this patch without the use of 'applypatch':
# STEP 1: Chdir to the source directory.
# STEP 2: Run the 'patch' program with this file as input.
#
#### End of Preamble ####

#### Patch data follows ####
diff -c "Task-Catalyst-4.00.orig\Makefile.PL" "Task-Catalyst-4.00\Makefile.PL"
Index: ./Makefile.PL
*** ./Makefile.PL	Sat Mar 20 01:30:29 2010
--- ./Makefile.PL	Thu Jul 15 23:10:11 2010
***************
*** 43,49 ****
                                          'Catalyst::Model::DBIC::Schema' => '0',
                                          'Catalyst::Model::Adaptor' => '0',
                                          'Catalyst::Component::InstancePerContext' => '0',
!                                         'FCGI::ProcManager' => '0',
                                          'CatalystX::REPL' => '0',
                                          'Catalyst::Plugin::Session::Store::DBIC' => '0',
                                          'FCGI' => '0',
--- 43,49 ----
                                          'Catalyst::Model::DBIC::Schema' => '0',
                                          'Catalyst::Model::Adaptor' => '0',
                                          'Catalyst::Component::InstancePerContext' => '0',
! #                                       'FCGI::ProcManager' => '0',
                                          'CatalystX::REPL' => '0',
                                          'Catalyst::Plugin::Session::Store::DBIC' => '0',
                                          'FCGI' => '0',
#### End of Patch data ####

#### ApplyPatch data follows ####
# Data version        : 1.0
# Date generated      : Thu Jul 15 23:11:20 2010
# Generated by        : makepatch 2.04
# Recurse directories : Yes
# Excluded files      : (\A|/).*\~\Z
#                       (\A|/).*\.a\Z
#                       (\A|/).*\.bak\Z
#                       (\A|/).*\.BAK\Z
#                       (\A|/).*\.elc\Z
#                       (\A|/).*\.exe\Z
#                       (\A|/).*\.gz\Z
#                       (\A|/).*\.ln\Z
#                       (\A|/).*\.o\Z
#                       (\A|/).*\.obj\Z
#                       (\A|/).*\.olb\Z
#                       (\A|/).*\.old\Z
#                       (\A|/).*\.orig\Z
#                       (\A|/).*\.rej\Z
#                       (\A|/).*\.so\Z
#                       (\A|/).*\.Z\Z
#                       (\A|/)\.del\-.*\Z
#                       (\A|/)\.make\.state\Z
#                       (\A|/)\.nse_depinfo\Z
#                       (\A|/)core\Z
#                       (\A|/)tags\Z
#                       (\A|/)TAGS\Z
# p "Makefile.PL" 3680 1279257011 0100666
#### End of ApplyPatch data ####

#### End of Patch kit [created: Thu Jul 15 23:11:20 2010] ####
#### Patch checksum: 55 2572 9092 ####
#### Checksum: 73 3273 1783 ####
